plugins/modules/pkgng.py

```diff
@@ -425,7 +439,8 @@ def main():
             rootdir=dict(required=False, type='path'),
             chroot=dict(required=False, type='path'),
             jail=dict(required=False, type='str'),
-            autoremove=dict(default=False, type='bool')),
+            autoremove=dict(default=False, type='bool'),
+            use_globs=dict(default=True, required=False, type='bool')),
```

Comment:
This way the next diff will look nicer when a new option is added:
```suggestion
            use_globs=dict(default=True, required=False, type='bool'),
        ),
```